packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -147,239 +105,463 @@ type TileProps = {
-                <Box
-                  className='iui-tile-thumbnail-picture'
-                  style={{ backgroundImage: `url(${thumbnail})` }}
-                />
-              ) : thumbnail && (thumbnail as JSX.Element).type === 'img' ? (
-                React.cloneElement(thumbnail as JSX.Element, {
-                  className: 'iui-tile-thumbnail-picture',
-                })
-              ) : (
-                <Box as='span' className='iui-thumbnail-icon'>
-                  {thumbnail}
-                </Box>
-              )}
-
-              {leftIcon &&
-                React.cloneElement(leftIcon as React.ReactElement, {
-                  className: 'iui-tile-thumbnail-type-indicator',
-                  'data-iui-size': 'small',
-                })}
-
-              {rightIcon &&
-                React.cloneElement(rightIcon as React.ReactElement, {
-                  className: 'iui-tile-thumbnail-quick-action',
-                  'data-iui-size': 'small',
-                })}
-
-              {badge && (
-                <Box className='iui-tile-thumbnail-badge-container'>
-                  {badge}
-                </Box>
-              )}
-            </Box>
-          )}
-
-          <Box className='iui-tile-content'>
-            {variant === 'folder' ? tileName : null}
-
-            {description != undefined && (
-              <Box className='iui-tile-description'>{description}</Box>
-            )}
-
-            {metadata != undefined && (
-              <Box className='iui-tile-metadata'>{metadata}</Box>
-            )}
-
-            {moreOptions && (
-              <DropdownMenu
-                onShow={showMenu}
-                onHide={hideMenu}
-                menuItems={(close) =>
-                  moreOptions.map((option: React.ReactElement) =>
-                    React.cloneElement(option, {
-                      onClick: (value: unknown) => {
-                        close();
-                        option.props.onClick?.(value);
-                      },
-                    }),
-                  )
-                }
-              >
-                <Box
-                  className={cx('iui-tile-more-options', {
-                    'iui-visible': isMenuVisible,
-                  })}
-                >
-                  <IconButton
-                    styleType='borderless'
-                    size='small'
-                    aria-label='More options'
-                  >
-                    <SvgMore />
-                  </IconButton>
-                </Box>
-              </DropdownMenu>
-            )}
-
-            {children}
-          </Box>
-
-          {buttons && <Box className='iui-tile-buttons'>{buttons}</Box>}
-        </Box>
-      </TileContext.Provider>
+      <Box
+        className={cx('iui-tile-thumbnail-picture', className)}
+        style={{
+          backgroundImage:
+            url && typeof url === 'string' ? `url(${url})` : undefined,
```

Comment:
`url &&` can be removed since there already is a check for if `url` exists

Reply:
I see. Removed.